apps/docs/src/components/landing/CTA.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button, H2, Prose } from '@uzh-bf/design-system'
+
+export function CTA() {
+  return (
+    <div className="space-y-4 py-16 text-center sm:py-24 md:space-y-8">
+      <H2 className={{ root: 'text-3xl' }}>Be Part of the Journey</H2>
```

Comment:
As we're restyling the entire landing page, could we maybe also try to improve the styling of these components, make them look more alike the rest and somewhat more interesting? :)